README.md

```diff
@@ -9,7 +9,7 @@ Sharable ESLint, Prettier and Rubocop configuration for AngelList.
 ```sh
 cd ../frontend-project
```

Comment:
I'm good with this, as long as it's getting fixed as part of automation should be fine. 

I do know there's a decent amount of violations to this rule so a quick post in fr-eng should be sufficient.